anyway/factories.py

```diff
@@ -0,0 +1,41 @@
+import factory
+
+from . import models
+from anyway.app_and_db import db
+
+class DefaultFactory(factory.alchemy.SQLAlchemyModelFactory):
```

Comment:
Is this file use for only  testing? - we might want to move it to a test folder

Reply:
correct, it's only for tests. I didn't put it under "tests" because it's not a test file, but I guess we can say it's sort of a configuration for tests, so I'll move it there, thanks